cav_msgs/msg/RoutePath.msg

```diff
@@ -0,0 +1,14 @@
+# This contains lanelet IDs of the route and the shortest path in the route
+
+# Timestamp and frame_id of the map
+Header header
+
+# Route name
+string route_name
+
```

Comment:
Can you add a route id field. Or do you feel different version of route generated due to a carma cloud update would get reflected in the name?

Reply:
Added. Those two should be separate field of a route msg.